dm/master/dm-master.toml

```diff
@@ -1,5 +1,10 @@
 # Master Configuration.
 
+# rpc configuration
+rpc-timeout = "30s"
+rpc-rate-limit = 10
+rpc-rate-burst = 40
```

Comment:
explain these two config items in comments?

Reply:
added in https://github.com/pingcap/dm/pull/157/commits/aab1ddce6c1bae083c51b638a908e2b0e5d91bf0